lib/graphql/schema/printer.rb

```diff
@@ -84,10 +84,62 @@ def print_deprecated(field_or_enum_value)
           end
         end
 
+        module DescriptionPrinter
+          def print_description(definition, indentation='', first_in_block=true)
+            return '' unless definition.description
+
+            lines = definition.description.split("\n")
+
+            description = indentation && !first_in_block ? "\n" : ""
```

Comment:
I've only noticed values of `""` and `"  "` for `indentation`, are there possible falsey values? If not, maybe this could be 

```ruby 
description = first_in_block ? "" : "\n"
```

?